### PR TITLE
Add request agents to newman run options

### PR DESCRIPTION
**Pull request by gone-skiing**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [ ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [ ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/postmanlabs/newman/pull/2392
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@gone-skiing thx for the update!

IMO it can be merged as is without making v4, the v5 breaking changes are not related to the API shape:
https://github.com/postmanlabs/newman/compare/v4.6.1...v5.0.0#diff-271a521af48d2d499403502d1c3b661d

**Comment by gone-skiing** [comment]
@peterblazejewicz could you clarify your comment? I am not sure what `without making v4` means in this context - my first rodeo in this repo.

**Comment by peterblazejewicz** [comment]
> could you clarify your comment? I am not sure what `without making v4` means in this context - my first rodeo in this repo.

TL;DR; You're good to go

Sure, the DT supports semver versioning, the package seems to introduce new major version. Usually one could create a v4 for the package and implement v5 (breaking changes) without wondering if the change will cause existing code to fail. But with those changes (at least from the release notes), there is no need for such steps I believe. It will just work. For more:
https://github.com/definitelyTyped/DefinitelyTyped/#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package

**Comment by gone-skiing** [comment]
Nice! Thanks for the clarification!

**Comment by gone-skiing** [comment]
Ready to merge
